components/layouts/ThePageFooter/FooterSection.vue

```diff
@@ -1,13 +1,21 @@
 <template>
   <section class="footer-section">
-    <h2 class="footer-section__title">
+    <h2
+      class="footer-section__title"
+      :class="{'footer-section__title_dark-mode': darkMode }"
```

Comment:
I think we can just use `footer-section__title--dark`, since it indicates the modifier anyway.

Reply:
We [decided](https://github.com/Qiskit/qiskit.org/issues/489) some months ago to use `_modifier` instead of `--modifier`, so we have the `_modifier` along all the code. We should continue doing the same to have a consistent code.

Also, only having `dark` I think is not appropiate because, for example, in this case, it makes me think that the `title` is dark, but the title is white on dark-mode. Do you know what I mean? What do you think? We can look for another name if you don't like `_dark-mode`, but I think it explains what it does

Reply:
sounds good... I thought I saw a `--purple` modifier at once, but I realize that might be from an override.